src/embed-code/components/EmbedContent.tsx

```diff
@@ -189,16 +205,25 @@ const EmbedContent: FC<EmbedProps & UserProps & EmbedFlowProps> = ({
 	};
 
 	const handleCopyButtonClicked = () => {
+		if (!savedEmbedCode) {
+			console.error('No embed code to copy');
```

Comment:
maybe add a toast to notify the user as well:
```suggestion
			console.error('No embed code to copy');
			ToastService.error(
				tHtml('Het kopieren van de embed code naar je klembord is mislukt.')
			);
```